core-libs/setup/ssr/logger/loggers/express-server-logger.ts

```diff
@@ -6,13 +6,15 @@
 
 import { InjectionToken } from '@angular/core';
 import { Request } from 'express';
+import { W3cTraceContext } from './w3c-trace-context/w3c-trace-context.model';
 
 /**
  * ExpressServerLoggerContext is used for log message in server side rendering.
  * It contains optional request object and additional properties that can be used in log message.
  */
 export interface ExpressServerLoggerContext {
   request?: Request;
+  traceContext?: W3cTraceContext;
```

Comment:
Is this addition really needed? Don't we pass only the express `Request` as a param to the ExpressServerLogger? e.g. `logger.log('some message', { request })`?

Reply:
N, we're passing the whole `ExpressServerLogger` object so I thought it would be nice to extend this interface with `traceContext` as it will be added by default if the header exists.
```
log(message: string, context: ExpressServerLoggerContext): void {
    /* eslint-disable-next-line no-console */
    console.log(this.stringifyWithContext(message, context));
  }
```